docs/tutorials/web/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploy on web
+parent: Tutorials
+has_children: true
+nav_order: 6
+---
+
+# How to add machine learning to your web application with ONNX Runtime
+
+ONNX Runtime Web enables you to run and deploy machine learning models in your web application using JavaScript APIs and libraries. This page outlines the general flow through the development process.
+
+To see an example of this flow in practice, you can follow the steps in the following tutorial to [build a web application to classify images using Next.js](classify-images-nextjs-github-template.md).
+
+For more detail on the steps below, see the [build a web application](../../reference/build-web-app.md) with ONNX Runtime reference guide.
+
+## ONNX Runtime web application development flow
+
+1. Choose deployment target and ONNX Runtime package
+
+   ONNX Runtime Web can be integrated into your web application in a number of different ways depending on the requirements of your application.
```

Comment:
not accurate. "ONNX Runtime Web"=>"ONNX Runtime libraries". Only the first scenario (Inference in browser) uses "ONNX Runtime Web". The other scenarios uses other libraries of ORT.

Reply:
There is another important scenario missing in this list: Electron.js

Electron is an important scenario, and we should mention it. (VS Code, for example, is built up using Electron)

ONNX Runtime Web may be useful in Electron based app, depending on the requirement

Reply:
Electron is mentioned further down in the paragraph.

Would the sentence be accurate if it was worded thus:

```
ONNX Runtime can be integrated into your web application in a number of different ways depending on the requirements of your application.
```

My goal is to keep this customer scenario focused.

Reply:
I'm thinking as I make these edits that it might be better to have a JavaScript developer page rather than this more generic one. But I think we should publish this and then move on to the improvements.